contracts/test/SequencerInbox.t.sol

```diff
@@ -19,24 +19,24 @@
 pragma solidity ^0.8.13;
 
 import "forge-std/Test.sol";
-import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
 import "../src/ISequencerInbox.sol";
 import "../src/libraries/Errors.sol";
+import {SpecularProxy} from "test/SpecularProxy.sol";
```

Comment:
just import `"@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol"`

Reply:
Done. Please review again.

Reply:
can you delete SpecularProxy now

Reply:
I am not sure what `forge diff` means?